components/ftx/common/pref_names.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_FTX_COMMON_PREF_NAMES_H_
+#define BRAVE_COMPONENTS_FTX_COMMON_PREF_NAMES_H_
+
+extern const char kFTXNewTabPageShowFTX[];
```

Comment:
We recommend to use `constexpr` like `constexpr char kFTXNewTabPageShowFTX[] = "ftx.new_tab_page.show_ftx";` in header file w/o cc.